src/main/java/entities/ChatRoom.java

```diff
@@ -0,0 +1,28 @@
+package entities;
+
+import java.util.ArrayList;
+
+public class ChatRoom {
+    ArrayList<Message> messages;
```

Comment:
I think the type here might better be List

Reply:
Yes, I think that would be more appropriate.